samcli/commands/build/core/options.py

```diff
@@ -76,6 +71,5 @@
             ),
         ],
     },
-    "Beta Options": {"option_names": {opt: {"rank": idx} for idx, opt in enumerate(BETA_OPTIONS)}},
-    "Other Options": {"option_names": {opt: {"rank": idx} for idx, opt in enumerate(OTHER_OPTIONS)}},
 }
+add_common_options_info(OPTIONS_INFO)
```

Comment:
How does this play with imports? there are places where OPTIONS_INFO is imported, does OPTIONS_INFO get mutated prior to import?

Reply:
When anything of the module or the module itself is imported, the entire file got executed, meaning that `add_common_options_info(OPTIONS_INFO)` is also executed when `samcli.commands.build.core.options` is imported.

I did a simple experiment to verify this behavior:
```
.
├── __init__.py
├── const.py
└── main.py
```

const.py:
```python
MY_DICT = {}

MY_DICT["foo"] = "bar"
```

main.py
```python
from const import MY_DICT

print(MY_DICT)
```

result:
```
» python main.py
{'foo': 'bar'}
```